R/pkg/R/group.R

```diff
@@ -242,18 +235,73 @@ createMethods()
 setMethod("gapply",
           signature(x = "GroupedData"),
           function(x, func, schema) {
-            try(if (is.null(schema)) stop("schema cannot be NULL"))
```

Comment:
this check of schema not being null still needs to be preserved for the the `gapply` call ?